src/source/Rtcp/RollingBuffer.c

```diff
@@ -0,0 +1,155 @@
+#define LOG_CLASS "RollingBuffer"
+
+#include "../Include_i.h"
+
+STATUS createRollingBuffer(UINT32 capacity, FreeDataFunc freeDataFunc, PRollingBuffer* ppRollingBuffer)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PRollingBuffer pRollingBuffer = NULL;
+    CHK(capacity != 0, STATUS_INVALID_ARG);
+
+    pRollingBuffer = (PRollingBuffer) MEMALLOC(SIZEOF(RollingBuffer) + SIZEOF(UINT64) * capacity);
```

Comment:
Mind doing `MEMCALLOC` instead of setting 0 explicitly, just in case new members get added.

Reply:
The performance is better on using MEMALLOC, given the rolling buffer could be very large, I would rather do MEMALLOC.